benefits/enrollment/views.py

```diff
@@ -26,11 +26,17 @@
 class IndexView(AgencySessionRequiredMixin, EligibleSessionRequiredMixin, RedirectView):
     """CBV for the enrollment landing page."""
 
+    route_origin = routes.ENROLLMENT_INDEX
+
+    def get_target_route_name(self):
+        return self.agency.enrollment_index_route
```

Comment:
Not the biggest deal, but it seems like `get_target_route_name` doesn't do much more / provide a lot of value over the existing `RedirectView.get_redirect_url`... could you just do this logic there? Then override `get_redirect_url` in the sub-class?

Reply:
Ah I agree, this is a nice simplification 👍 Thanks for the suggestion! Made the change in 35d16ca7d1970b995b395f6bfee5c8113e529d4f